agent/handlers/v1_handlers.go

```diff
@@ -65,12 +65,14 @@ func metadataV1RequestHandlerMaker(containerInstanceArn *string, cfg *config.Con
 }
 
 func newTaskResponse(task *api.Task, containerMap map[string]*api.DockerContainer) *v1.TaskResponse {
+	resp := &v1.TaskResponse{}
 	containers := []v1.ContainerResponse{}
 	for containerName, container := range containerMap {
 		if container.Container.IsInternal() {
 			continue
 		}
-		containers = append(containers, v1.ContainerResponse{DockerId: container.DockerID, DockerName: container.DockerName, Name: containerName})
+		containerResponse := newContainerResponse(containerName, container, task.GetTaskENI())
+		containers = append(containers, containerResponse)
```

Comment:
nit: would it be more readable to use `containerResponses` instead of `containers`?

Reply:
i decided against changing this because `containerResponse` and `containerResponses` are very similar